config/grunt/custom-tasks/github-pre-release.js

```diff
@@ -41,10 +41,12 @@ module.exports = function( grunt ) {
 
 			const pluginVersion = grunt.file.readJSON( "package.json" ).yoast.pluginVersion;
 
-			// For the first beta use the full changelog. For follow-up betas and RCs, only use the changes.
+			// For the first beta and RC, use the full changelog. For follow-up betas and RCs, only use the changes.
 			let initialContent = `Changes compared to ${grunt.config.get( "previousPluginVersion" )}:\n`;
-			if ( pluginVersion.substr( pluginVersion.length - 3 ) === "beta1" ) {
-				initialContent = `Changes in this beta (${pluginVersion}):\n`;
+
+			// Grab the string after index 5 (after "xx.x-") and check if it's the first beta or RC.
+			if ( pluginVersion.substr( 5 ) === "beta1" || pluginVersion.substr( 5 ) === "RC1" ) {
```

Comment:
this will break after 16.9-beta1 e.g. `16.10-beta1` returns `"-beta1"` which `!== "beta1"`
this is exactly why the original counted backwards from the end; and we can use framework code for that 🎉 
```suggestion
			if ( pluginVersion.endsWith( "beta1" ) || pluginVersion.endsWith( "RC1" ) ) {
```